docs/wiki/development/building.md

```diff
@@ -302,8 +305,11 @@ encouraged, and users should stick with leveraging the `package` target as detai
 To create a DEB, RPM, or TGZ on Linux, CPack will attempt to auto-detect the appropriate package type.
 You may override this with the CMake `PACKAGING_SYSTEM` variable as seen in the example below.
 
+Note: RPM will always try to create debuginfo packages, to do so though it needs the build folder
```

Comment:
It also needs the source folder to be longer than `/usr/src/debug/osquery/src_0`.